### PR TITLE
Add user

### PR DESCRIPTION
**Pull request by ArnoLC**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -78,6 +78,7 @@ GEM
       public_suffix (>= 2.0.2, < 6.0)
     autoprefixer-rails (10.4.13.0)
       execjs (~> 2)
+    bcrypt (3.1.18)
```
Comment (robq95) [inline Gemfile.lock:81]: Just another test :)

**Review by robq95 — COMMENTED** [review]
Added some test comments :)

File: Gemfile.lock (hunk 1 above)
Comment (ArnoLC) [reply Gemfile.lock:81]: Good test

Comment (robq95) [reply Gemfile.lock:81]: Solve it now :)

**Review by robq95 — APPROVED** [review]

**Pull request merged**